src/nodecontainer.js

```diff
@@ -164,6 +164,7 @@ NodeContainer.prototype.parseBackgroundSize = function(bounds, image, index) {
 
 NodeContainer.prototype.parseBackgroundPosition = function(bounds, image, index, backgroundSize) {
     var position = this.cssList('backgroundPosition', index);
+    var attachment = this.cssList('backgroundAttachment', index);
```

Comment:
Could you add some test for `background-attachment` as well, somewhere under https://github.com/niklasvh/html2canvas/tree/master/tests/cases/background